quantities/Extensions.cs

```diff
@@ -1,20 +1,19 @@
-using System.Globalization;
+﻿using System.Globalization;
 using Quantities.Measures;
-using Quantities.Numerics;
 
 namespace Quantities;
 
 public static class Extensions
 {
-  public static String ToString(this IFormattable formattable, String format) => formattable.ToString(format, CultureInfo.InvariantCulture);
-  internal static Quant To<TMeasure>(this in Double value)
-    where TMeasure : IMeasure
-    => Build<TMeasure>.With(in value);
-  public static void Serialize<TQuantity>(this IQuantity<TQuantity> quantity, IWriter writer)
-    where TQuantity : struct, IQuantity<TQuantity>, Dimensions.IDimension
-  {
-    writer.Start(typeof(TQuantity).Name.ToLowerInvariant());
-    quantity.Value.Write(writer);
-    writer.End();
-  }
+    public static String ToString(this IFormattable formattable, String format) => formattable.ToString(format, CultureInfo.InvariantCulture);
```

Comment:
I'm not sure if you know, but you can use [`dotnet format whitespace`](https://learn.microsoft.com/en-us/dotnet/core/tools/dotnet-format#whitespace) to fix all these formatting issues.

Reply:
Thanks for the hint :-)